src/modules/pokemons/PokemonHelper.ts

```diff
@@ -133,12 +133,32 @@ export function hasMegaEvolution(pokemonName: PokemonNameType): boolean {
     return !!P.pokemonMap[pokemonName].evolutions?.some((e) => e.restrictions.some((r) => r instanceof MegaEvolveRequirement));
 }
 
+export function hasUncaughtMegaEvolution(pokemonName: PokemonNameType): boolean {
+    return !!P.pokemonMap[pokemonName].evolutions?.some((e) => !App.game.party.alreadyCaughtPokemonByName(e.evolvedPokemon) && e.restrictions.some((r) => r instanceof MegaEvolveRequirement));
+}
+
 export function getMegaStones(pokemonName: PokemonNameType): MegaStoneItem[] {
     return GameHelper.enumStrings(MegaStoneType)
         .filter(s => (ItemList[s] as MegaStoneItem)?.basePokemon == pokemonName)
         .map(s => ItemList[s] as MegaStoneItem);
 }
 
+export function hasGigantamaxForm(pokemonName: PokemonNameType): boolean {
+    return P.pokemonMap['Gigantamax ' + pokemonName].id > 0 || P.pokemonMap['Eternamax ' + pokemonName].id > 0;
```

Comment:
Can you change the string concatenation here and in `hasUncaughtGigantamaxForm` to template literals instead? ESLint is supposed to catch these, not sure why it isn't